<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1303</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1281</width>
      <height>631</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1303</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionfonts"/>
    <addaction name="actionDarkMode"/>
    <addaction name="actionLightMode"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionLicense"/>
    <addaction name="actionHow_to_use"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionBold"/>
   <addaction name="actionitalic"/>
   <addaction name="actioncolorepicker"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionfonts">
   <property name="text">
    <string>fonts</string>
   </property>
  </action>
  <action name="actionDarkMode">
   <property name="text">
    <string>DarkMode</string>
   </property>
  </action>
  <action name="actionLightMode">
   <property name="text">
    <string>LightMode</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>License</string>
   </property>
  </action>
  <action name="actionHow_to_use">
   <property name="text">
    <string>How to use</string>
   </property>
  </action>
  <action name="actionAuto_Save">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Save</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset>
     <normaloff>src/icons/format_bold_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.svg</normaloff>
     <normalon>../../Downloads/format_bold_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.svg</normalon>src/icons/format_bold_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.svg</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionitalic">
   <property name="icon">
    <iconset>
     <normaloff>src/icons/format_italic_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.svg</normaloff>src/icons/format_italic_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.svg</iconset>
   </property>
   <property name="text">
    <string>italic</string>
   </property>
  </action>
  <action name="actioncolorepicker">
   <property name="icon">
    <iconset>
     <normaloff>src/icons/colors_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.svg</normaloff>src/icons/colors_24dp_1F1F1F_FILL0_wght400_GRAD0_opsz24.svg</iconset>
   </property>
   <property name="text">
    <string>colorepicker</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
